participants/philip-lenhart.json

```diff
@@ -0,0 +1,36 @@
+{
+    // required
+    "name": "Philip Lenhart",
+    // optional
+    "company": "optilyz GmbH",
+    // required
+    "when": {
+        "friday": true,
+        "saturday": true
+    },
+    // required
+    "iCanTakeNotesDuringSessions": true,
+    // required - at least one
+    "tags": ["TypeScript", "NodeJS", "AWS CDK", "React", "TDD", "Ensemble Programming"],
+    // optional
+    "vegan": false,
+    // optional
+    "vegetarian": false,
+    // optional
+    "allergies": [],
+    // required
+    "whatIsMyConnectionToJavascript": "For some years, I use JavaScript as well as TypeScript daily in my job",
+    // required
+    "whatCanIContribute": "Sharing knowledge around TypeScript, Ensemble Programming and CDK",
+    // optional
+    "tShirt": {
+        "size": "XL",
+        "type": "fitted"
+    },
+    // optional
+    "twitter": "philiplenhart",
+    // optional
+    "mastodon": "",
+    // optional
+    "website": ""
```

Comment:
Hi @Pipo93, thanks for your PR! 

Please just remove optional fields you do not want to set, because if you set them they have to be valid URLs in the case of website and mastodon and a non empty  array.

See also https://github.com/jscraftcamp/website/pull/1426/checks#step:9:17